substrate/frame/election-provider-multi-block/src/unsigned/miner.rs

```diff
@@ -1036,7 +1036,7 @@ mod trimming {
 						(40, Support { total: 40, voters: vec![(40, 40)] })
 					],
 					vec![
-						(30, Support { total: 11, voters: vec![(7, 7), (5, 2), (6, 2)] }),
+						(30, Support { total: 11, voters: vec![(5, 2), (6, 2), (7, 7)] }),
```

Comment:
these ordering changes are irrelevant -- the order within `Support` doesn't matter.